### ISSUE TITLE
Keyring information should not show up in transaction

### ISSUE THREAD

**Issue by jprupp** [jprupp/haskoin-core#230]
When getting a transaction using `gettx`, this information is displayed about the keyring:

``` json
{
    "created": "2016-01-24T21:18:36.797851Z",
    "mnemonic": null,
    "master": null,
    "name": "main"
}
```

The mnemonic is clearly not needed in this context. `master` is probably not needed either. In fact possibly the whole keyring section can be omitted.

**Comment by tphyahoo** [comment]
It seems to me there might be no harm stating which keyring/account a transaction is associated with.

I think keyrings in online wallets should be identified by nickname + xpub though, not mnemonic.  In my view, online wallets should not know mnemonics, nor any xprivs/privkeys.

**Comment by jprupp** [comment]
I agree. A fix is coming. For now this issue is fixed in #235
